src/lib.rs

```diff
@@ -0,0 +1,4590 @@
+    ///| to                | Email address of the recipient(s). Example: `"Bob <bob@host.com>"`. You can use commas to separate multiple recipients. Make sure to include all `To`, `Cc` and `Bcc` recipients of the message. |
+    ///| message           | MIME string of the message. Make sure to use `application/x-www-form-urlencoded` to send this as a file upload.                                                                                          |
+    ///| o:tag             | Tag string. See [Tagging](https://documentation.mailgun.com/en/latest/user_manual.html#tagging) for more information.                                                                                                                                            |
+    ///| o:deliverytime    | Desired time of delivery. See [Date Format](https://documentation.mailgun.com/en/latest/api-intro.html#date-format). Note: Messages can be scheduled for a maximum of 3 days in the future.                                                                        |
+    ///| o:dkim            | Enables/disabled DKIM signatures on per-message basis. Pass `yes` or `no`                                                                                                                    |
+    ///| o:testmode        | Enables sending in test mode. Pass `yes` if needed. See [Sending in Test Mode](https://documentation.mailgun.com/en/latest/user_manual.html#manual-testmode)                                                                                                               |
+    ///| o:tracking        | Toggles tracking on a per-message basis, see [Tracking Messages](https://documentation.mailgun.com/en/latest/user_manual.html#tracking-messages) for details. Pass `yes` or `no`.                                                                                              |
+    ///| o:tracking-clicks | Toggles clicks tracking on a per-message basis. Has higher priority than domain-level setting. Pass `yes`, `no` or `htmlonly`.                                                                 |
+    ///| o:tracking-opens  | Toggles opens tracking on a per-message basis. Has higher priority than domain-level setting. Pass `yes` or `no`.                                                                            |
+    ///| h:X-My-Header     | `h:` prefix followed by an arbitrary value allows to append a custom MIME header to the message (`X-My-Header` in this case). For example, `h:Reply-To` to specify Reply-To address.           |
+    ///| v:my-var          | `v:` prefix followed by an arbitrary name allows to attach a custom JSON data to the message. See [Attaching Data to Messages](https://documentation.mailgun.com/en/latest/user_manual.html#manual-customdata) for more information.                                         |
+    ///
+    ///Sends a `POST` request to `/{mydomain}/messages`
+    pub async fn send_messages<'a>(
```

Comment:
This is incorrect. It looks like this is the mime endpoint (`/<domain>/messages.mime`). Which also uses the unsupported `multipart/form-data` format. Let's replace this with tht eactual send message route (`POST /<domain>/messages`).